RPC/Sources/RPC/Handlers/ChainHandlers.swift

```diff
@@ -58,9 +58,12 @@ public enum ChainHandlers {
             self.source = source
         }
 
-        public func handle(request _: Request) async throws -> Response? {
-            // TODO: implement
-            nil
+        public func handle(request: Request) async throws -> Response? {
+            if let timeslot = request.value {
+                let block = try await source.getBlockHash(byTimeslot: timeslot)
+                return block.reduce(into: Data(capacity: block.count * 32)) { $0.append($1.data) }
```

Comment:
```suggestion
                let blocks = try await source.getBlockHash(byTimeslot: timeslot)
                return blocks.first
```

return any of of the hash is fine